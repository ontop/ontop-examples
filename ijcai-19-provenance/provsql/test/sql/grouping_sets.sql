\set ECHO none

SELECT city FROM personnel GROUP BY GROUPING SETS ((), (city));
