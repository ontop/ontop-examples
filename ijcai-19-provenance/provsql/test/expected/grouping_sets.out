\set ECHO none
ERROR:  GROUPING SETS, CUBE, and ROLLUP not supported by provsql
